README.md

```diff
@@ -90,23 +90,28 @@ Coming soon!
 
 ### Install PE
 
-1. Download the latest version of the PE installer for your platform
+1. Download the latest version of the PE installer for your platform.
  - https://puppetlabs.com/download-puppet-enterprise
 2. SSH into your Puppet master and copy the installer tarball into `/tmp`
-2. Expand the tarball and `cd` into the directory
-3. Run `puppet-enterprise-installer` to install
+2. Expand the tarball and `cd` into the directory.
+3. Run `puppet-enterprise-installer` to install PE.
```

Comment:
I'm not sure if they're still supposed to install monolithic (instead of split), if they are, tell them. If not, no problem.